packaging/windows/buildpackage.sh

```diff
@@ -49,97 +49,76 @@ BUILTDIR="${PACKAGING_DIR}/build"
 	echo "Output directory '${OUTPUTDIR}' does not exist.";
 	exit 1
 fi
 
-MOD_VERSION=$(grep 'Version:' mods/${MOD_ID}/mod.yaml | awk '{print $2}')
-
-if [ "${PACKAGING_OVERWRITE_MOD_VERSION}" == "True" ]; then
-	make version VERSION="${TAG}"
-else
-	echo "Mod version ${MOD_VERSION} will remain unchanged.";
-fi
-
-popd > /dev/null
-
 function build_platform()
 {
-	if [ "$1" = "x86" ]; then
-		IS_WIN32="WIN32=true"
+	PLATFORM="${1}"
+	if [ "${PLATFORM}" = "x86" ]; then
 		USE_PROGRAMFILES32="-DUSE_PROGRAMFILES32=true"
 	else
-		IS_WIN32="WIN32=false"
 		USE_PROGRAMFILES32=""
 	fi
 
-	pushd ${TEMPLATE_ROOT} > /dev/null
-
-	echo "Building core files ($1)"
-	pushd ${ENGINE_DIRECTORY} > /dev/null
-
-	SRC_DIR="$(pwd)"
+	if [ -n "${PACKAGING_DISCORD_APPID}" ]; then
+		USE_DISCORDID="-DUSE_DISCORDID=${PACKAGING_DISCORD_APPID}"
+	else
+		USE_DISCORDID=""
+	fi
 
-	make clean
-	make windows-dependencies "${IS_WIN32}"
-	make core "${IS_WIN32}"
-	make version VERSION="${ENGINE_VERSION}"
-	make install-engine gameinstalldir="" DESTDIR="${BUILTDIR}"
-	make install-common-mod-files gameinstalldir="" DESTDIR="${BUILTDIR}"
+	echo "Building core files (${PLATFORM})"
+	install_assemblies_mono "${TEMPLATE_ROOT}/${ENGINE_DIRECTORY}" "${BUILTDIR}" "win-${PLATFORM}" "False" "${PACKAGING_COPY_CNC_DLL}" "${PACKAGING_COPY_D2K_DLL}"
+	install_data "${TEMPLATE_ROOT}/${ENGINE_DIRECTORY}" "${BUILTDIR}"
 
 	for f in ${PACKAGING_COPY_ENGINE_FILES}; do
 		mkdir -p "${BUILTDIR}/$(dirname "${f}")"
-		cp -r "${f}" "${BUILTDIR}/${f}"
+		cp -r "${TEMPLATE_ROOT}/${ENGINE_DIRECTORY}/${f}" "${BUILTDIR}/${f}"
 	done
 
+	echo "Building mod files (${PLATFORM})"
+	pushd "${TEMPLATE_ROOT}" > /dev/null
+	make all
 	popd > /dev/null
 
-	echo "Building mod files ($1)"
-	make core
+	cp -Lr "${TEMPLATE_ROOT}/mods/"* "${BUILTDIR}/mods"
 
-	cp -Lr mods/* "${BUILTDIR}/mods"
+	for f in ${PACKAGING_COPY_MOD_BINARIES}; do
+		mkdir -p "${BUILTDIR}/$(dirname "${f}")"
+		cp "${TEMPLATE_ROOT}/${ENGINE_DIRECTORY}/bin/${f}" "${BUILTDIR}/${f}"
+	done
 
-	popd > /dev/null
+	set_engine_version "${ENGINE_VERSION}" "${BUILTDIR}"
+	if [ "${PACKAGING_OVERWRITE_MOD_VERSION}" == "True" ]; then
+		set_mod_version "${TAG}" "${BUILTDIR}/mods/${MOD_ID}/mod.yaml"
+	else
+		MOD_VERSION=$(grep 'Version:' "mods/${MOD_ID}/mod.yaml" | awk '{print $2}')
+		echo "Mod version ${MOD_VERSION} will remain unchanged.";
+	fi
 
 	# Create multi-resolution icon
 	convert "${ARTWORK_DIR}/icon_16x16.png" "${ARTWORK_DIR}/icon_24x24.png" "${ARTWORK_DIR}/icon_32x32.png" "${ARTWORK_DIR}/icon_48x48.png" "${ARTWORK_DIR}/icon_256x256.png" "${BUILTDIR}/${MOD_ID}.ico"
-	cp "${SRC_DIR}/OpenRA.Game.exe.config" "${BUILTDIR}"
-
-	# We need to set the loadFromRemoteSources flag for the launcher, but only for the "portable" zip package.
-	# Windows automatically un-trusts executables that are extracted from a downloaded zip file
-	cp "${SRC_DIR}/OpenRA.Game.exe.config" "${BUILTDIR}/${PACKAGING_WINDOWS_LAUNCHER_NAME}.exe.config"
```

Comment:
Removing this entirely will cause problems for itch deployments and portable installations.

Reply:
This is removing redundant code - `install_windows_launcher` handles this now.